src/components/SearchableSingleSelect/SearchableSingleSelect.tsx

```diff
@@ -180,14 +180,6 @@ export const SearchableSingleSelect = ({
                 />
             ))}
 
-            {hasSelectedInOptionList && selected && (
```

Comment:
this is to fix an unrelated bug (but noticed while working on this ticket). This code causes a second selection to appear checked at the bottom of SearchableSingleSelect.

When discussing, we thought this code was probably intended to be `!hasSelectedInOptionList` to show selection when a selection was not in the list of options, but this seems unnecessary as the UI component will show the non-list selection and when you click to change, we probably only want to show valid selections